networking_nsxv3/api/rpc.py

```diff
@@ -108,90 +81,46 @@ class NSXv3ServerRpcApi(object):
 
     rpc_version = nsxv3_constants.NSXV3_SERVER_RPC_VERSION
 
-    _LIMIT = 100
-    _CREATE_AFTER = datetime.utcfromtimestamp(0).isoformat()
-
-    def __init__(self, context, topic, host):
-        target = oslo_messaging.Target(topic=topic, version=self.rpc_version)
+    def __init__(self):
+        target = oslo_messaging.Target(topic=nsxv3_constants.NSXV3_SERVER_RPC_TOPIC, 
+                                       version=self.rpc_version)
+        self.context = neutron_context.get_admin_context()
         self.client = rpc.get_client(target)
-        self.context = context
-        self.host = host
-        self.topic = topic
+        self.host = cfg.CONF.host        
 
     @log_helpers.log_method_call
-    def get_port_revision_tuples(
-            self, limit=_LIMIT, created_after=_CREATE_AFTER):
+    def get_ports_revisions(self, limit, offset):
```

Comment:
I think, the function call should make clear, what the relationship between ports and revisions is.
E.g. `get_ports_with_revisions`, `get_ports_and_revisions` or `get_ports_by_revisions`.